market/dealmarket/deal_ingest_common.go

```diff
@@ -0,0 +1 @@
+package dealmarket
```

Comment:
rm?

Reply:
Yes, all the cleaning is still pending as stated in PR description. Did not want to drop files prematurely if this implementation doesn't make sense.